docs/TTS/GetSupportedLanguages/assets/GetSupportedLanguages.txt

```diff
@@ -1,44 +1,23 @@
 participant App
 participant SDL
 participant HMI
-
-HMI->SDL:BC.OnReady
+App->SDL:RegisterAppInterface
```

Comment:
The app is not related to this case, since there is no API which includes this information on the app side

Reply:
@jacobkeeler please see updates in ba730b1